counter/entrypoints/webapp.py

```diff
@@ -4,20 +4,28 @@
 
 from counter import config
 
-app = Flask(__name__)
-
-count_action = config.get_count_action()
-
-
-@app.route('/object-count', methods=['POST'])
-def object_detection():
-    uploaded_file = request.files['file']
-    threshold = float(request.form.get('threshold', 0.5))
-    image = BytesIO()
-    uploaded_file.save(image)
-    count_response = count_action.execute(image, threshold)
-    return jsonify(count_response)
-
-
+# Leo: I needed to instantiate in this method, so I can initiate in the tests.
```

Comment:
Yeah, this is good practice instead of having the app initialized at import time, you can remove this comment